src/django-app/django-project/quotes/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils.translation import gettext as _
+from shops.models import Shop
+
+class Quote(models.Model):
+    quote_request = models.OneToOneField("QuoteRequest", on_delete=models.CASCADE)
+    shop = models.ForeignKey(Shop, on_delete=models.CASCADE)
+    new_quote = 'NEW_QUOTE'
+    accepted = 'ACCEPTED'
+    in_progress = 'IN_PROGRESS'
+    done = 'DONE'
+    rework = 'REWORK'
+    status_choices = [
+        (new_quote, 'New Quote'),
+        (accepted, 'Accepted'),
+        (in_progress, 'In Progress'),
+        (done, 'Done'),
+        (rework, 'Rework')
+    ]
+    status = models.CharField(_("status"), max_length=12, choices=status_choices, default=new_quote)
```

Comment:
A better way to implement this would be:
```python3
    class Status(models.TextChoices):
        NEW_QUOTE = "new_quote", "New Quote"
        ACCEPTED = "accepted", "Accepted"
        ... (So on)
     status = models.CharField(_("status"), max_length=12, choices=Status.choices, default=new_quote)
```